packages/storage-plus/src/de.rs

```diff
@@ -134,7 +131,6 @@ impl<T: KeyDeserialize, U: KeyDeserialize> KeyDeserialize for (T, U) {
             value
                 .as_slice()
                 .try_into()
-                // FIXME: Add and use StdError try-from error From helper
```

Comment:
All these changes are the same lines:

```rust
u16::from_be_bytes(
             value
                 .as_slice()
                 .try_into()
                 .map_err(|err: TryFromSliceError| StdError::generic_err(err.to_string()))?,
         ) as usize;
```

Maybe they can be their own helper function as well, so we just call

`let t_len = parse_length(&value)?;`

Reply:
And once you do that, you can ignore the content of `err` and make it use case specific, like

```
.map_err(|_err| StdError::generic_err("Could not read 2 byte length"))?,
```

Reply:
That's not easy to do, because `parse_length` would have to be generic over the integers, and I don't have an associated trait for it.

Reply:
Already mentioned it in #432: https://github.com/CosmWasm/cw-plus/pull/432#discussion_r720134129

Reply:
But in this current diff it is always `u16::from_be_bytes`

Reply:
Ah, you're right! Will do.